velox/type/DecimalUtil.h

```diff
@@ -24,6 +25,11 @@
 
 namespace facebook::velox {
 
+enum class ConversionErrorCode : unsigned char {
+  INFINITE_FLOATING_NUMBER,
+  OVERFLOWED_RESCALED_VALUE,
```

Comment:
The error codes look restricted to rescaleDouble function. Do we consider make them more general for other functions to reuse?

Reply:
Yes. But do you think it's better to add other general fields in another pr?

Reply:
Can we make them more general to satisfy more cases? E.g.,
INFINITE_FLOATING_NUMBER -> INFINITY
OVERFLOWED_RESCALED_VALUE -> OVERFLOW.

Reply:
:eyes: I think maybe there are no differences. Perhaps we could discuss this in https://github.com/facebookincubator/velox/discussions/6323